src/App/Formatter/Post/CSV.php

```diff
@@ -103,11 +103,9 @@ public function createHeading($attributes)
 	 *
 	 * @return array
 	 */
-	protected function generateCSVRecords($records, $attributes)
+	public function generateCSVRecords($records, $attributes)
```

Comment:
Why public?

Reply:
my bad. I was wrangling with tests and forgot to change this back to protected. Thanks